Makefile

```diff
@@ -1,7 +1,13 @@
 build:
-	mv lib src
-	coffee -o lib -c src
+	cp -R lib src
+	coffee -c lib
+	find lib -iname "*.coffee" -exec rm '{}' ';'
 
 unbuild:
 	rm -rf lib
 	mv src lib
+
+publish:
```

Comment:
yeah, @hhsnopek we are moving away from having the `publish` task because if there's an error in publishing it can cause unexpected problems